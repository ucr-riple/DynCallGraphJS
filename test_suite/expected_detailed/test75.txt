{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test75.js:3:1:3:19)': [
        'max (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test75.js:3:1:3:19)'
    ]
}