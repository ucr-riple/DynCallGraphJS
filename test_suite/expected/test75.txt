{ '(test75.js:3:1:3:19)': [ 'max (Native)' ] }